client/lib/analytics/index.js

```diff
@@ -629,18 +641,21 @@ const analytics = {
 		initialized: false,
 
 		initialize: function() {
-			const parameters = {};
 			if ( ! analytics.ga.initialized ) {
-				if ( _user && _user.get() ) {
-					parameters.userId = hashPii( _user.get().ID );
-				}
-				window.ga( 'create', config( 'google_analytics_key' ), 'auto', parameters );
-				window.ga( function( tracker ) {
-					const clientId = tracker.get( 'clientId' );
-					window.ga( 'set', 'dimension3', clientId );
-				} );
-				window.ga( 'set', 'anonymizeIp', true );
-				window.ga( 'set', 'useAmpClientId', true );
+				const parameters = {
+					anonymize_ip: true,
+					transport_type: 'function' === typeof navigator.sendBeacon ? 'beacon' : 'xhr',
+					use_amp_client_id: true,
+					custom_map: {
+						dimension3: 'clientId',
```

Comment:
In Gtag, this is supposed to be `client_id` instead of `clientId`.
https://developers.google.com/analytics/devguides/collection/gtagjs/migration#client_id_and_user_id